File: tests/fixtures/swift-structure-proposals/overlay/Sources/SwiftA3Core/StructureEvidence/Omnibus.swift
public enum DomainOperations {
  public static func loadPayments() -> Int { 1 }
  public static func savePayments() -> Int { 2 }
  public static func loadCustomers() -> Int { 3 }
  public static func saveCustomers() -> Int { 4 }
  public static func loadExports() -> Int { 5 }
  public static func saveExports() -> Int { 6 }
  public static func renderExports() -> Int { 9 }
  public static func loadNotifications() -> Int { 7 }
  public static func saveNotifications() -> Int { 8 }
}

public func exportSurface() -> Int {
  DomainOperations.loadExports()
    + DomainOperations.saveExports()
    + DomainOperations.renderExports()
}

File: tests/fixtures/kotlin-structure-proposals/overlay/src/main/kotlin/cohort/Omnibus.kt
package cohort

fun loadPayments() = 1
fun savePayments() = 2
fun loadCustomers() = 3
fun saveCustomers() = 4
fun loadExports() = 5
fun saveExports() = 6
fun renderExports(): Int = loadExports() + saveExports()
fun loadNotifications() = 7
fun saveNotifications() = 8
